assets/sass/patterns/atoms/definition-list.scss

```diff
@@ -3,34 +3,71 @@
 .definition-list--timeline {
   dt,
   dd {
+    @include padding(28, "left");
     font-family: $font-secondary;
     font-weight: bold;
+    position: relative;
   }
 
   dt {
-    @include font-size-and-vertical-height(14, 24);
+    @include font-size-and-vertical-height(16, 24);
     @include margin(18, "top");
```

Comment:
Can we make this so the margin is bottom and not top.

Reply:
Discussed, we will keep the margin top because the toggle is optional and won't be visible when there is one element in the timeline.